tests/forms/gridfield/GridFieldLinkButtonTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class GridFieldLinkButtonTest extends SapphireTest
+{
+    protected static $fixture_file = 'GridFieldLinkButtonTest.yml';
+
+    public function setUp()
+    {
+        parent::setUp();
+    }
+}
```

Comment:
Please remove this class and GridFieldRefreshButtonTest, it's not adding any value.

If you're going to write some tests then it'd be great if you could match the folder structure, i.e. `Forms` over `forms` and implement namespaces as you go as well :)